portfolio/src/main/java/com/google/sps/controller/SimpleHtmlServlet.java

```diff
@@ -0,0 +1,25 @@
+package com.google.sps.controller;
+
+import java.io.*;
```

Comment:
It's typically recommended to not import `.*` and rather only the types you need. There are a number of reasons but this is the one I typically point to - https://stackoverflow.com/a/150315

Reply:
Thank you for letting me know🙏🙏🙏 I will improve this👍👍